netinterface.py

```diff
@@ -1,7 +1,9 @@
 # SPDX-License-Identifier: GPL-2.0-only
+import os
+import ipaddress
 
 import util
-import netutil
+import xcp.logger as LOG
```

Comment:
With the exception of `netutil.py`, the rest of the host installer code calls it `logger` rather than `LOG` so it might be nice to do the same for consistency (and fix `netutil.py` sometime in the future).